tests/data_access/models/test_todo_entry_model.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+import src.data_access.models.user_model as user_model
+from src.data_access.models.todo_entry import TodoEntry
+
+
+class TestSerialize:
+    def test_serialize_calls_to_nested_users_serialize(self, mocker):
+        mock_user = mocker.patch.object(user_model, "User")
+        mock_user.serialize.return_value = {}
+        expected_serialized_todo = {
+            "title": "title",
+            "id": None,
+            "date_time": datetime.now(),
+            "user": {},
```

Comment:
use mock_user.serialize.return_value, u dont have to set a return value unless the value is important to the test